### PR TITLE
ADD 10 CODECHEF PROBLEMS

### PR DESCRIPTION
**Pull request by aswanthkoleri**
ANKTRAIN-https://www.codechef.com/viewsolution/12834189
COLOR-https://www.codechef.com/viewsolution/12834130
FLOW009-https://www.codechef.com/viewsolution/12833699
KOL16H-https://www.codechef.com/viewsolution/12833664
LTM14AB-https://www.codechef.com/viewsolution/12834156
MOVIEWKN-https://www.codechef.com/viewsolution/12834075
PRB01-https://www.codechef.com/viewsolution/12833749
STICKS-https://www.codechef.com/viewsolution/12834106

### CODE REVIEW

File: CodeChef-Programs/ANKTRAIN.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include<stdio.h>
```
Comment (gauthamzz) [inline CodeChef-Programs/ANKTRAIN.c]: this program is too easy change this

**Comment by faheemzunjani** [comment]
@aswanthkoleri I think links to 2 problems are missing since you have given only 8 links. Otherwise looks good to me.

**Comment by aswanthkoleri** [comment]
https://www.codechef.com/viewsolution/12837284 - CHEFSTUD
https://www.codechef.com/viewsolution/12837311 - KOL16J

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Comment by aswanthkoleri** [comment]
there wasnt any difficulty level for the programs right??
if needed I can update more

**Comment by gauthamzz** [comment]
just change that program everything else is fine

**Comment by aswanthkoleri** [comment]
https://www.codechef.com/viewsolution/12841000 - LCOLLIS

**Comment by aswanthkoleri** [comment]
IS IT OKAY NOW ??

**Comment by faheemzunjani** [comment]
Looks good to me. Thanks! :)

**Pull request merged**